packages/nodes-base/credentials/HubspotOAuth2Api.credentials.ts

```diff
@@ -23,6 +23,7 @@ export class HubspotOAuth2Api implements ICredentialType {
 	extends = [
 		'oAuth2Api',
 	];
+	// eslint-disable-next-line n8n-nodes-base/cred-class-field-display-name-missing-oauth2
```

Comment:
1. I expect this exception is not needed, right? `displayName` here is compliant.
2. `displayName` in `HubspotDeveloperApi` (not this cred) should get flagged, does it not? It should be either fixed or excepted. (Unfamiliar with Hubspot to say which.)

Reply:
My bad, I added it to the wrong hubspot cred, time for a small coffee break...

Reply:
Fixed